src/NUnitTestAdapter/Properties/AssemblyInfo.cs

```diff
@@ -21,5 +21,5 @@
 [assembly: ComVisible(false)]
 
 [assembly: Guid("c0aad5e4-b486-49bc-b3e8-31e01be6fefe")]
-[assembly: AssemblyVersion("3.5.0")]
-[assembly: AssemblyFileVersion("3.5.0")]
+[assembly: AssemblyVersion("3.5.1")]
```

Comment:
It may confuse things to change the Assembly version without changing the package versions to match. However, I can fix this up when doing the release.